### PR TITLE
there is no g_bot_repair cvar

### PR DESCRIPTION
**Pull request by bmorel**

### CODE REVIEW

**Comment by slipher** [comment]
Also update the ckit option description to mention repairing?

**Comment by bmorel** [comment]
Fun fact: there is no such entry.

File: ui/server_bots.rml [hunk 1]
```diff
@@ -50,13 +50,13 @@
...
 				</row>
-				<row>
-					<input cvar="g_bot_repair" type="checkbox" />
-					<h3> Human bots repair buildings </h3>
-				</row>
 
 				<h2> Individually </h2>
 				<h3> Primary weapons </h3>
+				<row>
+					<input cvar="g_bot_" type="checkbox" />
```
Comment (slipher) [inline ui/server_bots.rml:57]: wrong cvar name

Comment (bmorel) [reply ui/server_bots.rml:57]: what the... well, obviously a wrong cvar name...

File: ui/server_bots.rml [hunk 2]
```diff
@@ -50,13 +50,13 @@
...
 				</row>
-				<row>
-					<input cvar="g_bot_repair" type="checkbox" />
-					<h3> Human bots repair buildings </h3>
-				</row>
 
 				<h2> Individually </h2>
 				<h3> Primary weapons </h3>
+				<row>
+					<input cvar="g_bot_ckit" type="checkbox" />
```
Comment (slipher) [inline ui/server_bots.rml:57]: Bad news... the g_bot_ckit cvar is actually unused

**Comment by slipher** [comment]
LGTM

**Pull request merged**